devconf-outreachy/source/index.rst

```diff
@@ -0,0 +1,344 @@
+
+Becoming OpenStack Developer via Outreachy
+==========================================
+
+Presented by: Aija Jauntēva, Dmitry Tantsur, Richard Pioso, Ilya Etingof
+
+In this talk
+------------
```

Comment:
@sagesharp Hey, we are planning a presentation on our past Outreachy internship experience. I wonder if you would be able to take a look at the slides/notes...? Your feedback and advice would be much appreciated!